.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,34 @@
-> Please provide us with the following information:
-> ---------------------------------------------------------------
-
-### OS?
-> Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
-
-
+<!--
+IF YOU DON'T FILL OUT THE FOLLOWING INFORMATION YOUR ISSUE MIGHT BE CLOSED WITHOUT INVESTIGATING
+-->
+### Bug Report or Feature Request (mark with an `x`)
+- [ ] bug report -> please search issues before submitting
+- [ ] feature request
+
+ <!-- if bug report, otherwise delete -->
 ### Versions.
-> Please run `ng --version`. If there's nothing outputted, please run in a Terminal: `node --version` and paste the result here:
+<!--
+Output from: `ng --version`.
+If nothing, output from: `node --version` and include OS details.
+  Windows (7/8/10). Linux (incl. distribution). macOS (El Capitan? Sierra?)
+-->
 
 
+ <!-- if bug report, otherwise delete -->
 ### Repro steps.
-> Was this an app that wasn't created using the CLI? What change did you do on your code? etc.
+<!-- Steps to reproduce this bug. -->
```

Comment:
I disagree, the point of this exercise is to make the template less intimidating hopefully getting more people to use the template to help us out.